src/commands/app/db/collection/validate.js

```diff
@@ -106,7 +106,11 @@ ValidateCollection.args = {
 }
 
 ValidateCollection.flags = {
-  ...DBBaseCommand.flags
+  // Inherit all DB base flags except region
```

Comment:
Why is region a DB base flag at all?  I think it should be moved from BaseCommand to StateBaseCommand, the only db command that uses it is provision.